policy_definitions/network_security/policy_set_definition_qby_NetworkSecurity.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "QBY-NetworkSecurity",
+    "type": "Microsoft.Authorization/policySetDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "description": "Enforce namingconform NSG to have a rule that denys all inbound traffic in Vnet. Ensures all subnets and NSG to be namingconform and with their Vnet in name associated",
+        "displayName": "NetworkSecurity",
+        "metadata": {
+            "version": "0.1.0",
```

```suggestion
            "version": "1.0.0",
```
We don't really use that tag but for conformity we should set it to 1.0.0